fetch.bs

```diff
@@ -1123,6 +1123,56 @@ format of range header value can be set using <a>add a range header</a>.
 <a>implementation-defined</a> <a for=/>header value</a> for the `<code>User-Agent</code>`
 <a for=/>header</a>.
 
+<p>To
+<dfn export for="header list" id=concept-header-list-filter-include>filter include</dfn> a
+<a for=/>header list</a> <var>list</var> and <a for=/>header name</a>s <var>includeNames</var>, run these steps:
```

Comment:
The second argument here is a list of header names? As such this is not a correct type. (Also applicable below.)